docker/Dockerfile.ubuntu

```diff
@@ -0,0 +1,92 @@
+# This file aimed for developers and optimized for quick re-builds
+# Shouild be used by running `make localdocker`
+FROM ffmpeg-base:nvidia as builder
+
+FROM golang:1-stretch as builder2
+ENV PKG_CONFIG_PATH /root/compiled/lib/pkgconfig
+WORKDIR /root
+RUN apt update \
+    && apt install -y \
+    git gcc g++ gnutls-dev 
+    # git gcc g++ gnutls-dev linux-headers
+COPY --from=builder /root/compiled /root/compiled/
+#COPY --from=builder /usr/local/cuda /usr/local/cuda
+
+ENV PKG_CONFIG_PATH /root/compiled/lib/pkgconfig
+#ENV CPATH /usr/local/cuda/include 
```

Comment:
Is this commented out for when we use CUDA drivers?

Reply:
this is still a work in progress, the flag wasn't needed at that point. I'll remove it 👍